[AN-861] Minor improvement. Example walkthrough how to connect to the database. (#53)
aws/amazon-neptune-jdbc-driver: markdown/sql.md

Each Builder method listed in the documentation is accepted as a property. For example, "**[port](https://tinkerpop.apache.org/javadocs/current/full/org/apache/tinkerpop/gremlin/driver/Cluster.Builder.html#port-int-)**(int port)" means that as a part of the connection string it may be configured as a property or added as a Key-Value Pair in Properties. See below for examples.

#### Establishing Connection Manually
 
Amazon Neptune Database is a VPC only service, and you need to establish an SSH tunnel before connecting to the database outside of the VPC (e.g. from the local machine).
To establish an SSH tunnel, you need an EC2 instance deployed in the same VPC as Neptune Database. More information on how to configure SSH Tunnel is described in [Connecting to Neptune](/markdown/setup/configuration.md#using-an-ssh-tunnel-to-connect-to-amazon-neptune)

> - Step 1. Establish Port Forwarding through SSH Tunnel

Reviewer: Nit: Use numbered list without step numbers instead of bullets.

Author: Changed

> - Step 2. Add Neptune Database hostname in /etc/hosts (see [Adding SSH Tunnel Lookup](/markdown/setup/configuration.md#adding-a-ssh-tunnel-lookup) for more details).

Reviewer: Windows doesn't have /etc/hosts. Should add windows and Mac.

Ultimately would like to remove this requirement asap

Author: Rephrased. Will not be duplicting instructions.

> - Step 3. Enter Connection String. As an example in [DBVisualizer](https://www.dbvis.com/) enter *Database URL* `jdbc:neptune:sqlgremlin://<hostname>;port=8182;authScheme=NONE`. If you are using default port and using IAM athentication you do not need to supply addiitonal parameters.

Reviewer 2: I think <hostname> and port here are tied to the SSH tunnel that you set up, is that correct? If so, can you note that?

Author: Fixed

4. Now you can connect to your Amazon Neptune Database instance.

> 

Reviewer: Nit:
```suggestion
```

#### Establishing Connection Manually
 
Amazon Neptune Database is a VPC only service, and you need to establish an SSH tunnel before connecting to the database outside of the VPC (e.g. from the local machine).
To establish an SSH tunnel, you need an EC2 instance deployed in the same VPC as Neptune Database. More information on how to configure SSH Tunnel is described in [Connecting to Neptune](/markdown/setup/configuration.md#using-an-ssh-tunnel-to-connect-to-amazon-neptune).

>  1. Establish Port Forwarding through SSH Tunnel

Reviewer 2: ```suggestion
1. Establish Port Forwarding through SSH Tunnel
```

> 2. Add Neptune Database hostname in `hosts` (see [Adding SSH Tunnel Lookup](/markdown/setup/configuration.md#adding-a-ssh-tunnel-lookup) for more details how to add configuraiton in Windows and MacOS).

Reviewer 2: ```suggestion
2. Add Neptune Database hostname in `hosts` (see [Adding SSH Tunnel Lookup](/markdown/setup/configuration.md#adding-a-ssh-tunnel-lookup) for more details how to add configuration in Windows and macOS).
```

> 3. Enter Connection String. As an example in [DBVisualizer](https://www.dbvis.com/) enter *Database URL* `jdbc:neptune:sqlgremlin://<hostname>;port=<port>;authScheme=NONE`. Default *port* is `8182`, however you can configure port forwarding to any available port on your machine. If you are using default port and using IAM athentication you do not need to supply addiitonal parameters.

Reviewer 2: ```suggestion
3. Enter Connection String. As an example in [DBVisualizer](https://www.dbvis.com/) enter *Database URL* `jdbc:neptune:sqlgremlin://<hostname>;port=<port>;authScheme=NONE`. Default *port* is `8182`, however you can configure port forwarding to any available port on your machine. If you are using default port and using IAM authentication you do not need to supply additional parameters.
```

Reviewer 2: What should <hostname> be, what should port be, based on the SSH tunnel that you've established?
